src/index.tsx

```diff
@@ -3,14 +3,28 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
-import { ChakraProvider } from '@chakra-ui/react';
+import { ChakraProvider, extendTheme } from '@chakra-ui/react';
+
+const theme = extendTheme({
+  semanticTokens: {
+    colors: {
+      darkRed: "#430E0E",
+      lightRed: "#9A5757",
+      darkPink: "#FFBEBE",
+      lightPink: "#FFE2E2",
+      lightYellow: "#FFC700",
+      darkGrey: "#1F1F1F",
+      lightGrey: "#B7B7B7"
```

Comment:
close enough, so i'll approve. but on here, https://chakra-ui.com/docs/styled-system/customize-theme notice instead of color name, it's naming it something like 'brand' or 'primary' or 'secondary'.

Reply:
I started out doing that, but then there were too many colors to come up with original names for that were memorable enough to know which color I was referring to.